src/bun.js/builtins/cpp/BundlerPluginBuiltins.cpp

```diff
@@ -327,11 +327,27 @@ const char* const s_bundlerPluginRunSetupFunctionCode =
     "  };\n" \
     "\n" \
     "  var setupResult = setup({\n" \
+    "    config,\n" \
     "    onDispose,\n" \
     "    onEnd,\n" \
     "    onLoad,\n" \
     "    onResolve,\n" \
     "    onStart,\n" \
+    "    //\n" \
+    "    initialOptions: {\n" \
+    "      bundle: true,\n" \
+    "      entryPoints: config.entrypoints ?? config.entryPoints,\n" \
```

Comment:
if the user messes this up, they'll get a weird error in the plugin instead of a more helpful error saying this field is missing

i suggest `?? []` at the end

Reply:
![image](https://github.com/oven-sh/bun/assets/24465214/5c5d6ed1-70b4-4743-9bc7-771c72379ace)
i know what you mean but that error doesn't happen. ill add it anyways since it should definitely exist.